source/features/hidden-review-comments-indicator.css

```diff
@@ -1,4 +1,4 @@
-.rgh-comments-indicator {
+:root .rgh-comments-indicator {
```

Comment:
This was the only fix required to make the button visible again.

The reason why the line isn't being shown is because GitHub now uses `::before` to show the actual line, so our code fighting with GitHub’s own style.

An easier solution would be to either:

- only use ::after, which doesn't seem to be used by GitHub
- ensure that all the CSS actually applies OR figure out why it doesn't

I undid the previous changes because I don’t think they were going in the right direction for the most part. I'd be ok merging this as is for now and the line can be restored later.